cudamapper/src/overlapper.cpp

```diff
@@ -191,8 +230,9 @@ void Overlapper::print_paf(const std::vector<Overlap>& overlaps, const std::vect
                     overlap.target_length_,
                     overlap.target_start_position_in_read_,
                     overlap.target_end_position_in_read_,
-                    overlap.num_residues_,
-                    0,
+                    overlap.num_residues_ * k, // Print out the number of residue matches multiplied by kmer size to get approximate number of matching bases
+                    std::max(std::abs(static_cast<int>(overlap.target_start_position_in_read_) - static_cast<int>(overlap.target_end_position_in_read_)),
```

Comment:
Can you specify the number of bits here? As it depends of the number of bits of `position_in_read_t` your safest bet is probably to go for `std::int64_t`.
That's the reason I was suggesting adding a function that does this conversion. In that case we could use `std::make_signed()` to ensure the number of bits stays the same:
https://github.com/clara-genomics/ClaraGenomicsAnalysis/blob/8ad6a5d3d3707a351a558d60138ddc07ec76839c/common/utils/include/claragenomics/utils/signed_integer_utils.hpp#L25

Reply:
Yes, since it is CPU code, I would just use `int64_t` here.

Reply:
done